### PR TITLE
feat(graphical-selector): add fullscreen functionality

### PR DESCRIPTION
**Pull request by xincczn**

### CODE REVIEW

File: frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx [hunk 1]
```diff
@@ -224,6 +226,10 @@ const GraphicalSelector = () => {
     graph.zoomTo(zoom - getZoomRatio());
   };
 
+  const toggleSidebar = () => {
+    setSidebar(!sidebar);
```
Comment (Peedee2002) [inline frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx:230]:
```suggestion
    setSidebar(sidebar => !sidebar);
```
to make sure that all input is taken

File: frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx [hunk 2]
```diff
@@ -254,28 +263,32 @@ const GraphicalSelector = () => {
...
-          </Button>
-          <Button onClick={handleHideGraph}>
-            Hide Graph
-          </Button>
-          <div>
-            {course ? <div>{course.code} - {course.title}</div> : 'No course selected'}
-          </div>
-        </S.SidebarWrapper>
+        {sidebar
+          ? (
```
Comment (Peedee2002) [inline frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx]: isnt this better expressed as sidebar && bla?

**Review by Peedee2002 — APPROVED** [review]
A few small comments but overall good

File: frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx (hunk 1 above)
Comment (xincczn) [reply frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx:230]: wait can you elaborate on this

**Pull request merged**